frontend/lib/identification/qr_code_scanner/qr_code_scanner.dart

```diff
@@ -21,11 +24,14 @@ class QrCodeScanner extends StatefulWidget {
 
 class _QRViewState extends State<QrCodeScanner> {
   final MobileScannerController _controller = MobileScannerController(
-    torchEnabled: false,
-    detectionSpeed: DetectionSpeed.normal,
-    formats: [BarcodeFormat.qrCode],
-    returnImage: false,
-  );
+      torchEnabled: false, detectionSpeed: DetectionSpeed.normal, formats: [BarcodeFormat.qrCode], returnImage: false);
+  final MobileScannerController _controllerCameraResolution = MobileScannerController(
```

Comment:
not sure how i can avoid defining different controllers twice since `cameraResolution` has no setter to change it